cmd/crossplane/main.go

```diff
@@ -59,7 +65,11 @@ func main() {
 		extCmd = app.Command("stack", "Perform operations on stacks")
 
 		// stack manage - adds the stack manager controllers and starts their reconcile loops
-		extManageCmd = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageCmd                     = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageTenantKubeconfig        = extManageCmd.Flag("tenant-kubeconfig", "The absolute path of the kubeconfig file to Tenant Kubernetes instance. Setting this will activate host aware mode of Stack Manager").String()
```

Comment:
I think if you use `.ExistingFile()` instead of `.String()` Kingpin will ensure the referred path (if any) is an existing file.